pkgs/development/tools/rust/cargo-nextest/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, Security
+, libiconv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-nextest";
+  version = "0.9.9";
+
+  src = fetchFromGitHub {
+    owner = "nextest-rs";
+    repo = "nextest";
+    rev = "cargo-nextest-${version}";
+    sha256 = "sha256-1s1N126S51kg7aOgAb8oMts1zJcO6QRn1fwbQf6ZaJ8=";
+  };
+
+  cargoSha256 = "sha256-JxZyl5Hti3Hh33e7H/pXhM6WkU0kDDml0naBPYzvNy4=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+    libiconv
+  ] ++ lib.optionals stdenv.isDarwin [
+    Security
+  ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "A next-generation test runner for Rust.";
```

```suggestion
    description = "Test runner for Rust";
```
[No period](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) at the end. The removal of the "marketing blabla" is opinionated.

---

> [No period](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) at the end.

Thanks, missed that :)

> The removal of the "marketing blabla" is opinionated.

Haha sure, I simply copied it from their `Cargo.toml`. I'll shorten here :)